src/main/resources/css/common.css

```diff
@@ -159,12 +168,28 @@ span {
         bottom: 0;
         left: 0;
     }
+    .navbar__item {
+        display: flex;
+        flex-direction: column;
+    }
+    .navbar__item--active {
+        border-top: 2px solid rgb(1, 90, 217) !important;
+        border-bottom: none;
+    }
 }
 /* --mobile layout for phones  */
 @media only screen
 and (max-device-width: 767px)
 and (min-device-width: 320px) {
- .hero__page-title {
+    .navbar__item {
+        display: flex;
+        flex-direction: column;
+    }
+    .navbar__item--active {
+        border-top: 2px solid rgb(1, 90, 217) !important;
+        border-bottom: none;
+    }
```

Comment:
these `.navbar__item` and `.navbar__item--active` inside the media query are identical to the ones above from 171 - 178. If you aren't overwriting any styling or changing anything inside the media query then you shouldn't need the second set of the styles. so you can delete the ones inside the media query

Reply:
yes I removed that, I thought it was different thing until I notice the max-device-width which applies for both phone & tablet